Cython/Compiler/ModuleNode.py

```diff
@@ -1592,13 +1593,23 @@ def generate_new_function(self, scope, code, cclass_entry):
             code.putln("new((void*)&(p->%s)) %s();" % (
                 entry.cname, decl_code))
 
+        if scope.is_internal == 2 and py_attrs:
```

Comment:
I have a feeling that `is_x` should only be `True` or `False` and if we want to represent more states then it should not be `is_x`

Reply:
Yes that's probably right. I'll replace it with a better name at the next round of revisions (once I've thought of a better name..)